org.musescore.MuseScore.yaml

```diff
@@ -28,8 +28,9 @@ finish-args:
   # Allow loading/saving files from anywhere
   # (portals don’t work yet)
   - --filesystem=host
-  # Allow other instances to see lockfiles
-  - --env=TMPDIR=/var/tmp
```

Comment:
Why was this removed?

Reply:
For
```
QtSingleCoreApplication: listen on local socket failed, QLocalServer::listen: Name error
```

This actually delayed the startup.

Reply:
Better question: Does single instance work after this removal?

Reply:
I have no idea how to test this.

Reply:
launch the application twice..?

Reply:
ah right. and it fails. I'll figure something out. (#50 doesn't include this)

Reply:
Removed this....